public/css/site.css

```diff
@@ -41,6 +41,9 @@
 .st-default-search-input{
 	color: #000000 !important; 
 }
+.st-default-search-input:focus {
+  border-color: #3a98e3 !important;
```

Comment:
super-nit: Could've matched the indent of the other `.st-default-search-input` styles :-)